scalding-serialization-macros/src/main/scala/com/twitter/scalding/serialization/macros/impl/ordered_serialization/TreeOrderedBuf.scala

```diff
@@ -166,19 +178,25 @@ object TreeOrderedBuf {
         case m: MaybeLengthCalculation[_] =>
           val tmpLenRes = freshT("tmpLenRes")
           q"""
-            @inline def noLenCalc = {
-              $noLenCalc
-            }
             @inline def withLenCalc(cnt: Int) = {
```

Comment:
Could do it either way, this hasn't really changed in this RB. I'm not really sure making this inline was a wonderful idea to begin with. just easier to inline than put it as its own private function right now. 

Context: In XProf for large objects seeing paths not being JIT'd probably from methods growing too large. The thrift variants bail out and don't fully recurse. for case classes/maps we do allow recursion though